components/layout/OpenAuthoringSiteForm.tsx

```diff
@@ -43,14 +44,23 @@ const OpenAuthoringSiteForm = ({
     return undecorateSaveListener
   }, [form])
 
+  useEffect(() => {
+    const failedListener = createDecorator({
+      afterSubmitFailed: () => setStatefulPreviewError('Failed to save data. Your fork may have been deleted.')
```

Comment:
This could go in the same useEffect as the one above:
```
    const submitListener = createDecorator({
      afterSubmitSucceeded: () =>
        cms.alerts.success(
          `Saved Successfully: Changes committed to ${Cookies.get(
            'fork_full_name'
          )}`
        ),
afterSubmitFailed: //...
    })
```